docs/ConfigureScalarDL.md

```diff
@@ -18,8 +18,14 @@ You need to create a secret object with `db-username` and `db-password` to store
     # username=<AWS_ACCESS_KEY_ID>
     # password=<AWS_ACCESS_SECRET_KEY>
 
+# AWS RDS Databases(MySQL PostgreSQL, Oracle, SQL Server, Aurora), Azure Databases for MySQL, PostgreSQL
```

Comment:
I think we can use the same keyword used in schema loader for a standard
https://github.com/scalar-labs/scalardb/tree/master/schema-loader

Reply:
Updated changes similar to schema loader section. But used USERNAME instead of USER as it suited better(Also there were already some discrepancies with schema loader configuration with this document).